assets/css/styles.css

```diff
@@ -460,3 +491,1037 @@ a {
 .business__btn.btn--black .btn__icon {
   background-color: rgba(255, 255, 255, .1)
 }
+
+/* Project */
+.project-section {
+  margin-bottom: 120px;
+  position: relative;
+}
+
+@media screen and (max-width: 767px) {
+  .project-section {
+    margin-bottom: 60px;
+  }
+}
```

Comment:
Gom nhóm các css reponsive 767 của components project lại 1 chỗ. Chỗ này a thấy e đang lặp code @media screen and (max-width: 767px)